README.md

```diff
@@ -0,0 +1,60 @@
+# CoDoc19 :robot:
+
+An open source bot which helps to diagnose symptoms of Covid 19 virus staying at home!
+
+<img src="https://user-images.githubusercontent.com/15226353/77837846-5e04ff00-718b-11ea-89b7-a3f0cefa25c8.jpg" data-canonical-src="https://user-images.githubusercontent.com/15226353/77837846-5e04ff00-718b-11ea-89b7-a3f0cefa25c8.jpg" width="250" height="500"  />
+
+## Usage
+Currently we have telegram support. So just search CoDoc 19 in telegram and start interacting with it :man_technologist:
```

Comment:
Bot's telegram handler is `@CoDoc19Bot`. Can you add it ?

Reply:
added it. :+1: